client/js/ifml/fromjson/intermediate/elementrules.js

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) 2018, the IFMLEdit.org project authors. Please see the
+// AUTHORS file for details. All rights reserved. Use of this source code is
+// governed by a MIT-style license that can be found in the LICENSE file.
+/*jslint node: true, nomen: true */
+"use strict";
+
+var almost = require('almost'),
+    createRule = almost.createRule;
+
+module.exports = [
+    createRule(
+        function (element, model) {
+            return model.isPositionedElement(element);
+        },
+        function (element) {
+            return {
+                elements: {
```

Comment:
You can directly return the _object_ itself or an _array_ of _objects_ if you need to generate more than one.